src/main/java/org/example/ngevaticketmanagerspring/user/User.java

```diff
@@ -0,0 +1,66 @@
+package org.example.ngevaticketmanagerspring.user;
+
+import java.time.LocalDate;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+import jakarta.validation.constraints.Pattern;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
```

Comment:
```suggestion
@Data
@NoArgsConstructor
@Entity
@Table(name = "user")
```

1. Durch die einmalige Annotation @Data (aus Lombock) müssen nirgends mehr @Getter- und @Setter-Annotationen gesetzt und kein Konstruktor mit Parametern mehr geschrieben werden.
https://projectlombok.org/features/Data

2. Der Konstruktur ohne Parameter kann durch die Lombock-Annotation @NoArgsConstructor ersetzt werden.
3. Im ELE2 sind die Entitäten in der Tabelle alle in der Einzahl benannt. Also besser "user" statt "users"